teacher.rb

```diff
@@ -0,0 +1,11 @@
+require './person'
+class Teacher < Person
+  def initialize(_specialization, age, name, parent_permission)
```

Comment:
- [ ] in this class the `name` and `parent_permission` should be assigned a default value as well.